labgrid/driver/serialdriver.py

```diff
@@ -27,12 +26,6 @@ class SerialDriver(ConsoleExpectMixin, Driver, ConsoleProtocol):
         bindings = {"port": "SerialPort", }
     else:
         bindings = {"port": {"SerialPort", "NetworkSerialPort"}, }
-    if version.parse(serial.__version__) != version.Version('3.4.0.1'):
-        message = ("The installed pyserial version does not contain important RFC2217 fixes.\n"
-                   "You can install the labgrid fork via:\n"
-                   "pip uninstall pyserial\n"
-                   "pip install https://github.com/labgrid-project/pyserial/archive/v3.4.0.1.zip#egg=pyserial\n")  # pylint: disable=line-too-long
-        warnings.warn(message)
```

Comment:
> labgrid's pyserial fork is now referenced in the dependencies, so there
> should be no scenario left where the user needs to intervene manually.

This is only correct when using pip to install the package. If a user relies on a distribution (perhaps Debian/Ubuntu or OE), this can still fail.

Reply:
True, I didn't think of this case. I'll leave it there and update the latest version.

Reply:
Done.